plugin/src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/rest/RestPPLQueryAction.java

```diff
@@ -197,6 +198,8 @@ public void onFailure(Exception e) {
     ResponseFormatter<QueryResult> formatter;
     if (format.equals(Format.CSV)) {
       formatter = new CsvResponseFormatter(pplRequest.sanitize());
+    } else if (format.equals(Format.RAW)) {
+      formatter = new RawResponseFormatter(pplRequest.sanitize());
```

Comment:
Should we let the raw format always be sanitized instead?

Reply:
I recall as discussed we need to separate sanitize and escape logic in `FlatResponseFormatter`.

1. For CSV, always do escape for `,`, sanitize if parameter `sanitize` given
2. For RAW, always do escape for `|`, but **never sanitize**. Because: a) that sanitize logic only make sense for Excel; b) assumption is that users want the data exported as much original as possible.

Reply:
@dai-chen implemented "For RAW, always do escape for |, but never sanitize. "